cli/admin/src/main/java/io/pravega/cli/admin/dataRecovery/DurableDataLogRepairCommand.java

```diff
@@ -0,0 +1,914 @@
+            case "StreamSegmentSealOperation":
+                segmentId = getLongUserInput("Introduce Segment Id for StreamSegmentSealOperation:");
+                result = new StreamSegmentSealOperation(segmentId);
+                offset = getLongUserInput("Introduce Segment Offset for StreamSegmentSealOperation:");
+                ((StreamSegmentSealOperation) result).setStreamSegmentOffset(offset);
+                break;
+            case "StreamSegmentTruncateOperation":
+                segmentId = getLongUserInput("Introduce Segment Id for StreamSegmentTruncateOperation:");
+                offset = getLongUserInput("Introduce Offset for StreamSegmentTruncateOperation:");
+                result = new StreamSegmentTruncateOperation(segmentId, offset);
+                break;
+            case "UpdateAttributesOperation":
+                segmentId = getLongUserInput("Introduce Segment Id for UpdateAttributesOperation:");
+                result = new UpdateAttributesOperation(segmentId, createAttributeUpdateCollection());
+                break;
+            default:
+                output("Invalid operation, please select one of " + operations);
+                throw new UnsupportedOperationException();
+        }
+        return result;
+    }
+
+    /**
+     * Provides two ways of creating the payload of {@link Operation}s with binary content (MetadataCheckpointOperation,
+     * StorageMetadataCheckpointOperation, StreamSegmentAppendOperation): i) zero, which means to provide a content of
+     * a defined length consisting of just 0s, ii) file, which will read the contents of a specified file and use it as
+     * payload for the {@link Operation}.
+     *
+     * @return Binary contents for the {@link Operation}.
+     */
+    @VisibleForTesting
+    ByteArraySegment createOperationContents() {
+        ByteArraySegment content = null;
+        do {
+            try {
+                switch (getStringUserInput("You are about to create the content for the new Operation. " +
+                        "The available options are i) generating 0s as payload (zero), ii) load the contents from a provided file (file): [zero|file]")) {
+                    case "zero":
+                        int contentLength = getIntUserInput("Introduce length of the Operation content: ");
+                        content = new ByteArraySegment(new byte[contentLength]);
+                        break;
+                    case "file":
+                        String path = getStringUserInput("Introduce the path for the file to use as Operation content:");
+                        content = new ByteArraySegment(Files.readAllBytes(Path.of(path)));
+                        break;
+                    default:
+                        output("Wrong option. Please, select one of the following options: [zero|file]");
+                }
+            } catch (Exception ex) {
+                outputError("Some problem has happened.");
```

Comment:
I can see some unexpected error getting the user into some infinite loop with no way to get out. Maybe we should provide an option to abort?

Reply:
Added possibility to abort this process.